Dockerfile

```diff
@@ -1,33 +1,21 @@
-FROM alpine/git:v2.30.2 AS getter
 
-RUN mkdir /pygeofilter
-WORKDIR /pygeofilter
-# switch to geopython/pygeofilter once bug fixes are merged via PR
-RUN git clone --single-branch --branch main https://github.com/sparkgeo/pygeofilter.git
-WORKDIR /pygeofilter/pygeofilter
-RUN git checkout 5cfa8afbb6ffc8a30b8dde5be7a2981d4b534678
+FROM python:3.8-slim
 
+RUN apt-get update && apt-get install -y \
+      # OS dependencies only required to build certain Python dependencies
+      libpq-dev \
+      python3-pip \
+      python3-psycopg2 \
+      # OS dependency required at runtime
+      curl \ 
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Decided to combine all of the dependencies (build and runtime) here for now. Created an issue to enhance the dockerfiles to benefit from multistage build in the future: #19